### ISSUE TITLE
Add Dark mode in the app.

### ISSUE THREAD

**Issue by Skynet843** [DSCKGEC/Galleriz#6]
### Here is the reference.
<img src="https://user-images.githubusercontent.com/49488404/180698413-31d35207-ddb7-4acc-a24d-f926e76bdbab.jpeg" alt="demo" width="30%"/>

**Comment by Pransav290302** [comment]
I would like to add that dark mode.

**Comment by Pransav290302** [comment]
Please assign to me this task.

**Comment by Skynet843** [comment]
Do you have registered in KSOC22

**Comment by Pransav290302** [comment]
Yes @Skynet843 I have registered in KSOC22 and also selected this  Galleriz project in that form.

**Comment by Skynet843** [comment]
Ok, I assign it to you. Ping me on discord if need any help.
